src/constants/captions_da.js

```diff
@@ -60,6 +60,7 @@ export default {
     writtenby: 'av',
     toteacherinstruction: 'Til lærerveiledning',
     tolesson: 'Til oppgave',
+    reset: 'Start på nytt',
```

Comment:
Hva med "Fjern avkrysninger"? Tooltip: "Fjern alle avkrysninger for denne oppgaven".